### DISCUSSION TITLE
Readme needs be fix

### DISCUSSION THREAD

**Discussion by DumiJDev** [gluonhq/scenebuilder#669] [General]
I found in README a wrong path and i should like fix it

![image](https://github.com/gluonhq/scenebuilder/assets/72232653/9762ea43-e765-4f6f-9501-143d24ae39e6)

If it is not a mistake, please tell me.

**Comment by Oliver-Loeffler** [comment]
What should be there instead?

**Reply by DumiJDev** [reply]
![image](https://github.com/gluonhq/scenebuilder/assets/72232653/d3538afd-690e-46c1-9f35-84ca3bb9200f)

**Comment by Oliver-Loeffler** [comment]
I have to apologize, havent spotted it - now I see it. Sorry. Yes.

**Comment by DumiJDev** [comment]
Can I fix it?

**Reply by Oliver-Loeffler** [reply]
Well of course. Please sign the contributor agreement with Gluon. 

All details are described here: https://github.com/gluonhq/scenebuilder#issues-and-contributions

**Comment by Oliver-Loeffler** [comment]
The issue has been fixed. Thanks!
